src/item.cpp

```diff
@@ -3661,6 +3661,10 @@ bool item::can_contain( const itype &tp ) const
 
 bool item::spill_contents( Character &c )
 {
+    if( is_container_empty() || !is_container() ) {
```

Comment:
Fair enough but I'd invert the order of the conditional - it both reads better that way and maybe more efficient